Applications/Spreadsheet/CellTypeDialog.cpp

```diff
@@ -403,4 +425,90 @@ CellTypeMetadata CellTypeDialog::metadata() const
     return metadata;
 }
 
+ConditionView::ConditionView(ConditionalFormat& fmt)
+    : m_format(fmt)
+{
+    load_from_json(cond_fmt_view_ui_json);
+    static_cast<GUI::Label*>(find_descendant_by_name("if_label"))->set_text("if...");
+    static_cast<GUI::Label*>(find_descendant_by_name("fg_color_label"))->set_text("Foreground Color...");
+    static_cast<GUI::Label*>(find_descendant_by_name("bg_color_label"))->set_text("Background Color...");
```

Comment:
You can set these in the JSON :^)

Reply:
Unfortunately, `text` does not apply to `GUI::Label`, and registering it made the linker unhappy.
I kinda forgot about that one actually ^^'

Reply:
a5878175e25bf28265d1358c2fdb1cc4b66cc61e ^_^

Reply:
oh.
I'll change it lol